lib/manager/components/ProjectFeedListToolbar.js

```diff
@@ -288,6 +307,46 @@ export default class ProjectFeedListToolbar extends PureComponent<Props> {
     return options
   }
 
+  _renderLabelFilter = () => {
+    const {
+      project,
+      filter
+    } = this.props
+    // This variable must be set directly from this.props both
+    // to allow the override and for flow
+    const labels = this.props.filter.labels || []
+    const { labelsFilterMode } = filter
+
+    return (
+      <MenuItem header style={{ minWidth: 275 }}>
+        <h5>
+          Show{' '}
+          <select
+            name='anyonly'
+            id='anyonly'
+            value={labelsFilterMode}
+            onChange={this._onLabelModeSelectorClick}
+          >
+            <option value='any'>any</option>
+            <option value='only'>only</option>
+          </select>{' '}
+          feeds with the labels
+        </h5>
+        <div className='feedLabelContainer'>
+          {project.labels.map((label) => (
+            <FeedLabel
```

Comment:
As mentioned elsewhere, I would prefer the checkbox to be separated out from the label (to the left) so that we have a more consistent visual language with respect to the labels.

Reply:
@landonreed is your above comment regarding text flow consistent with your other comment https://github.com/ibi-group/datatools-ui/pull/693#discussion_r677445990?

Reply:
I believe so, feel free to raise any specific inconsistencies, @binh-dam-ibigroup.

Reply:
Have moved the checkboxes out, and it feels consistent with the changes I made to the edit/delete buttons. I've changed the labels to be as you say, but there's (to my eyes) some awkwardness now with the button placement. But that may just be me. Let me know what you think! @landonreed @binh-dam-ibigroup